src/pages/TripDetailPage/TripsDetailPage.styled.ts

```diff
@@ -0,0 +1,3 @@
+import styled from "styled-components";
+
+export const TripDetailPageWrapper = styled.div``;
```

Comment:
use html semantic elements

Reply:
Which one?

Reply:
This is meant to be just a placeholder element for now.

What is the best for a placeholder element? Any suggestions other than `div`?

Reply:
if you are suggesting that it is a wrapper containing all the individual trip details, isnt it better to make it a section rather than a div.

Reply:
@nijin-26 @abhijith-biju What do you think about `section` for wrapper elements?

Reply:
> @nijin-26 @abhijith-biju What do you think about `section` for wrapper elements?

As Lekshmi have told us to use more semantic elements. So we thought of using `Section` for wrapper or containers and div or any other semantics inside them.

Reply:
Okay. Will update.

Reply:
Updated in last commit